### ISSUE TITLE
FR: Release request. Provide APK as a Download possibilty on owncloud/download

### ISSUE THREAD

**Issue by Kawohl** [owncloud/android#2074]
Hey Guys as sombeody suggested in FB :
![bildschirmfoto 2017-12-14 um 09 18 36](https://user-images.githubusercontent.com/12275313/33982497-06de769c-e0b1-11e7-8763-ec36ceb141cf.png)
What do you think? Should we add that as an deliverable or is their anything to consider that would be against this?

**Comment by michaelstingl** [comment]
Our official Android app is available in Google Play Store. User always have the latest version with Play Store Updates ==> best user experience for unexperienced users.

Users that install the *.apk don't get auto updates of the app. Also, to install the *apk directly, users would need to disable security features on their devices. This is nothing we should recommend.

Unofficial *.apk's are available on other places, F-Droid for example. Everybody that is interested should easily find it.

**Comment by Kawohl** [comment]
Yes but for instance there is no way to install on amazon devices without apks


john@owncloud.com
-----------------------------
Jonathan Kawohl

Tel: +49 151 637 49 502

owncloud.com
ownCloud GmbH, GF: Tobias Gerlinger, Holger Dyroff, Harald Weimer
Rathsbergstr. 17, 90411 Nürnberg, HRB 28050 (AG Nürnberg)

**Comment by michaelstingl** [comment]
We could point to it from a less prominent position. (FAQ section in docs?)

**Comment by Kawohl** [comment]
Good Point! 

john@owncloud.com
-----------------------------
Jonathan Kawohl

Tel: +49 151 637 49 502

owncloud.com
ownCloud GmbH, GF: Tobias Gerlinger, Holger Dyroff, Harald Weimer
Rathsbergstr. 17, 90411 Nürnberg, HRB 28050 (AG Nürnberg)

**Comment by jesmrec** [comment]
Already available in [Releases](https://github.com/owncloud/android/releases) section
